### ISSUE TITLE
Feature request: docker-compose

### ISSUE THREAD

**Issue by hyper-ex** [boredazfcuk/docker-icloudpd#1]
Just as the title says, support for docker-compose launching.

**Comment by boredazfcuk** [comment]
I don't use docker-compose, but I've just had a quick glance at it and from what I can tell, docker-compose is an application for defining the services/links/dependencies between multi-container environments.

I'm not sure what benefits docker-compose will offer as this is only a single container? With that in mind, and looking at a few examples, it's .yaml file would be equivalent to just copy/pasting the **docker create \\** command into a iCloudPD.sh executable script.

**Comment by boredazfcuk** [comment]
I've spent a few hours looking at this today. I do now use docker-compose to manage this container and I'll admit it is easier to keep up to date.

However, as my container is designed to be run multiple times, it makes docker-compose inflexible as most of the information need to be hard-coded (hostname, container name, volume names, paths, environment file etc).

I've created a generic docker-compose.yaml file and example env_file which people can use as a base to create their own customised version. I'll probably not be updating it too frequently though as it differs so much from my 'production' version.
